test/unittest/dds/subscriber/DataReaderTests.cpp

```diff
@@ -2421,15 +2421,6 @@ TEST_F(DataReaderUnsupportedTests, UnsupportedDataReaderMethods)
         ReturnCode_t::RETCODE_UNSUPPORTED,
         data_reader->get_matched_publication_data(publication_data, publication_handle));
 
-    {
-        SampleStateMask sample_states = ANY_SAMPLE_STATE;
-        ViewStateMask view_states = ANY_VIEW_STATE;
-        InstanceStateMask instance_states = ANY_INSTANCE_STATE;
-        EXPECT_EQ(
-            nullptr,
-            data_reader->create_readcondition(sample_states, view_states, instance_states));
```

Comment:
Remove create_readcondition from the list of unimplemented methods in this test comment.

Reply:
👍